scripts/accumulator-server.py

```diff
@@ -266,6 +290,7 @@ def record_request(request):
 
     global ac, t0, times
     s = ''
+    times = []
```

Comment:
Maybe this change is related with the one in L495 and L553? Could you elaborate a little bit on it, please?

Reply:
Well, the function `record_request` appends data to a list called `times`, so the list must be declared locally inside the function.

Reply:
Currently times is used at:

* record_request() and on_message() to append data
* times_() to return its content
* number() to return its len
* reset() to reset it

So, it is actually a global variable shared by these functions. I see that `global` is used in record_request(), on_message() and reset(). Maybe it should be added to times_() and number()?

Apart from that , the commented lines at L495 and L553 should be removed, to avoid leaving leftover comments in the code.

Moreover, I think ac and t0 setting at L551-L552 can be removed, as this is already done at L493-L494. What do you think?

Reply:
Sure, I didn't noticed that either, the point is you need to use `global` locally when you need to perform operations on some variable, but it's not apparently neccesary to include the `global` statement when you are just reading them.